docs/en/supported_models/supported_models.md

```diff
@@ -2,37 +2,42 @@
 
 ## Models supported by TurboMind
 
-|       Model        |   Size   | FP16/BF16 | KV INT8 | W4A16 |
-| :----------------: | :------: | :-------: | :-----: | :---: |
-|       Llama        | 7B - 65B |    Yes    |   Yes   |  Yes  |
-|       Llama2       | 7B - 70B |    Yes    |   Yes   |  Yes  |
-|      InternLM      | 7B - 20B |    Yes    |   Yes   |  Yes  |
-|     InternLM2      | 7B - 20B |    Yes    |    -    |  Yes  |
-| InternLM-XComposer |    7B    |    Yes    |   Yes   |  Yes  |
-|        QWen        | 7B - 72B |    Yes    |   Yes   |  Yes  |
-|      QWen-VL       |    7B    |    Yes    |   Yes   |  Yes  |
-|      Baichuan      |    7B    |    Yes    |   Yes   |  Yes  |
-|     Baichuan2      |    7B    |    Yes    |   Yes   |  Yes  |
-|     Code Llama     | 7B - 34B |    Yes    |   No    |  No   |
-|         YI         | 6B - 34B |    Yes    |   No    |  No   |
+|       Model        |    Size    | FP16/BF16 | KV INT8 | W4A16 |
+| :----------------: | :--------: | :-------: | :-----: | :---: |
+|       Llama        |  7B - 65B  |    Yes    |   Yes   |  Yes  |
+|       Llama2       |  7B - 70B  |    Yes    |   Yes   |  Yes  |
+|      InternLM      |  7B - 20B  |    Yes    |   Yes   |  Yes  |
+|     InternLM2      |  7B - 20B  |    Yes    |    -    |  Yes  |
+| InternLM-XComposer |     7B     |    Yes    |   Yes   |  Yes  |
+|        QWen        |  7B - 72B  |    Yes    |   Yes   |  Yes  |
```

Comment:
Shall we also update this? https://github.com/InternLM/lmdeploy/issues/1008

Reply:
OK